FALSEHOODS.md

```diff
@@ -118,3 +118,9 @@ surprising how many false assumptions programmers continue to make about them.
 
     In Egypt, it is common for phone numbers to be written in native digits.
 
+1.  **The leading zero in domestic numbers can always be discarded when dialing
```

Comment:
This is a good one! Maybe we should order it after "To make a number dialable, you only need to change the prefix" since it's also about dialling prefixes?

Also, perhaps substitute "domestic numbers" with "numbers formatted for domestic usage" and change "The leading zero" to "A leading zero" since there may not be one?